### ISSUE TITLE
DJ Streamer Does not connect

### ISSUE THREAD

**Issue by infotodoporjesus** [AzuraCast/AzuraCast#6936]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #ab9b626 (2024-02-10 2:39)

### What happened?

Hello!
The WebDJ is not working.
Does not connect. DJ Streamer username and password are added, I click on the start connection button, it does not connect.
(Translated with Google Translate)

**Comment by Moonbase59** [comment]
I don’t normally use that but can confirm. No connect and nothing to see in LS or Icecast logs.

**Comment by infotodoporjesus** [comment]
I tried Firefox and Brave browsers.
First with Shoutcast and then with Icecast server.

**Comment by infotodoporjesus** [comment]
Hello!
I asked someone else if I had that problem.
Tested on 5 stations and the Web Dj tool, it worked perfectly on Azuracast version 0.19.4.
How can I install that version of Azuracast again?

**Comment by itachipirate** [comment]
brand new install on docker, also having the same issue. This is the firefox browser log when I click "start streaming" in the web dj
![githubwebdjproblem](https://github.com/AzuraCast/AzuraCast/assets/89754955/3d3bb805-0d6f-430c-b296-1989171f8c91)

**Comment by ThaMunsta** [comment]
I tested Mixxx and that worked but yeah the WebDJ on my docker install doesnt work. Different error though. I get:
`Failed to construct 'WebSocket': The URL 'undefined' is invalid.`

**Comment by alexaungmyooo** [comment]
```
DOMException: Failed to construct 'WebSocket': The URL 'undefined' is invalid.
    at connect (https://radio.thitsarparami.org/static/vite_dist/assets/PublicWebDJ-D7lpc58V.js:1:8272)
    at startStream (https://radio.thitsarparami.org/static/vite_dist/assets/PublicWebDJ-D7lpc58V.js:1:3389)
    at x.onClick.s.<computed>.s.<computed> (https://radio.thitsarparami.org/static/vite_dist/assets/PublicWebDJ-D7lpc58V.js:1:18541)
    at Ue (https://radio.thitsarparami.org/static/vite_dist/vue-D_WLdR4x.js:13:1373)
    at we (https://radio.thitsarparami.org/static/vite_dist/vue-D_WLdR4x.js:13:1443)
    at HTMLButtonElement.n (https://radio.thitsarparami.org/static/vite_dist/vue-D_WLdR4x.js:17:8113)
```

Web DJ base in application error

**Comment by jvsouto** [comment]
WebDJ on my docker install doesnt work. The same error occurs on the demo page https://demo.azuracast.com

`DOMException: Failed to construct 'WebSocket': The URL 'undefined' is invalid.`

**Comment by phonic2k** [comment]
I also get the same after updating to 19.5 on 3 installs with webDJ:

vue-D_WLdR4x.js:13 DOMException: Failed to construct 'WebSocket': The URL 'undefined' is invalid.

**Comment by BusterNeece** [comment]
Thank you all for the reports. Should be fixed in the latest Rolling Release, which will be incorporated into the next Stable release.

**Comment by alexaungmyooo** [comment]
Rolling Release #[ca425d3](https://github.com/AzuraCast/AzuraCast/commit/ca425d3e9cab18c42ef7b2da26b5e09d13093ab8) (2024-02-20 11:03) • Docker • PHP 8.3

I still got same issue. I have tested with both Firebox and Chrome

**Comment by BusterNeece** [comment]
@alexaungmyooo Your Rolling Release version is older than the version where it was fixed, which was published the same time as my comment above. Please update before re-testing.

**Comment by ThaMunsta** [comment]
Awesome!! I "merged" in the code from commit b83d54b and it's working for me. 44.1kHz had some performance issues on the machine I just tested with but that's a me issue to solve since 32kHz is fine. 😬 
Thanks all ❤️

**Comment by infotodoporjesus** [comment]
Thank God!
Thank you Azuracast Team for responding to our requests!
Waiting for stable update!
(Translated with Google translator)
